index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
My Comment

Reply:
The problem here is that too many Tetris pieces are the same color. I suggest that you choose colors which would be easier to see when playing the game.